docs/src/pages/components/rating/react.mdx

```diff
@@ -12,224 +27,225 @@ Rating shows users a star rating of a product or service (currently read-only).
 
 Import Rating component and styles
 
-```jsx
-import { Rating } from '@aws-amplify/ui-react';
-import '@aws-amplify/ui-react/styles.css';
+<Example>
+  <DefaultRatingExample />
+  <ExampleCode>
+    ```jsx file=./examples/DefaultRatingExample.tsx
 
-<Rating />;
-```
+    ```
 
-<Example>
-  <Rating />
+  </ExampleCode>
 </Example>
 
 ### Rating Value
 
 Pass in a `value` prop to set the Rating value.
 
-```jsx
-<Rating value={3.7} />
-```
-
 <Example>
-  <Rating value={3.7} />
+  <RatingValueExample />
+  <ExampleCode>
+    ```jsx file=./examples/RatingValueExample.tsx
+
+    ```
+
+  </ExampleCode>
 </Example>
 
 ### Max Value
 
 Pass in a `maxValue` prop to set the maximum value of the Rating component.
```

Comment:
Is this a required prop, or does it default to 5? I think it's worth mentioning in the docs.